Documentation/trying-out-rkt.md

```diff
@@ -121,7 +121,7 @@ UUID		APP	IMAGE NAME					STATE	CREATED		STARTED		NETWORKS
 0c3ab969	nginx	registry-1.docker.io/library/nginx:latest	running	2 minutes ago	2 minutes ago	default:ip4=172.16.28.2
 ```
 
-The nginx container is now accessible on the host under http://172.16.28.2.
+From the above list we can see, that the nginx container got the IP address `172.16.28.2` assigned (may vary on your system). Since we established a route from the host to the `172.16.28.0/24` pod network the nginx container is now accessible on the host under http://172.16.28.2.
```

Comment:
In this example, the nginx container was assigned the IP address `172.16.28.2` (the address assigned on your system may vary).